docker-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Add logstash as command if needed
+if [[ "$1" == -* ]]; then
+	set -- logstash "$@"
+fi
+
+# Run as user "logstash" if the command is "logstash"
+if [[ "$1" == logstash ]]; then
```

Comment:
Could we skip the special behavior of `[[` here and just use `[ "$1" = logstash ]` instead? :+1: